docs/section5/Rule5-36.md

```diff
@@ -2,7 +2,7 @@
 # Envelope - Rule 5-36  
 
 **Rule ID:** 5-36  
-**Rule Description:**  Skylight area must be allocated to surfaces in the same proportion in the baseline as in the proposed design; Skylight orientation and tilt shall be the same as in the proposed design.
+**Rule Description:**  Skylight area must be allocated to surfaces in the same proportion in the baseline as in the proposed design.
```

Comment:
Line 5 needs an extra space character at the end to force a line break. See the rendered html.